src/igvfd/schemas/changelogs/model_set.md

```diff
@@ -2,6 +2,8 @@
 
 ### Minor changes since schema version 4
 
+* Add `donors`
+* Add `samples`.
```

Comment:
The properties were already there, just not submittable. I think something like Allow submission of `samples` or something like that would be better.

Reply:
yes will do, was wondering the same thing just didn't find a previous example.